tests/unit/components/enhanced-table/filter-checkbox/FilterCheckbox.spec.jsx

```diff
@@ -0,0 +1,44 @@
+import { render, fireEvent } from '@testing-library/react'
+import { vi } from 'vitest'
+
+import FilterCheckbox from '~/components/enhanced-table/filter-checkbox/FilterCheckbox'
+
+let mockedFilter = []
+let mockedCheckbox
+const mockedSetFilter = vi.fn()
+
+const mockedFilterCheckbox = {
+  label: 'adminTable.active',
+  value: 'active'
+}
+
+describe('FilterCheckbox test', () => {
+  it('should add checkboxValue to filters when is checked', () => {
+    const { getByLabelText, rerender } = render(<FilterCheckbox filter={ mockedFilter } filterCheckbox={ mockedFilterCheckbox } setFilter={ mockedSetFilter } />)
```

Comment:
try to render it once in beforeEach block and use screen.getByLabelText from '@testing-library/react'

Reply:
i tried so but method getByLabelText imported from '@testing-library/react' didn't work properly and couldn't find checkbox by this label

Reply:
also entered prop filter in FilterCheckbox component have to be different in two test cases so i render it in every case